README.md

```diff
@@ -93,6 +93,23 @@ and whether they worked (✅) or did not (❌):
 | Kind                          | 1.23.5             | ❌            | ✅          |
 | OpenShift                     | 1.23.3             | ✅            | ✅          |
 
+## Running on KinD cluster
+
+### How to run on kind cluster
+
+1. Download latest [KinD](https://kind.sigs.k8s.io/docs/user/quick-start) stable version
```

Comment:
this is already part of the `make prereqs` target right ?

Reply:
yes good point we can do what u suggested below